.travis.yml

```diff
@@ -13,7 +13,7 @@ rust:
 - nightly
 - beta
 - stable
-- 1.40.0
+- 1.42.0
```

Comment:
1.40 is explicitly the minimum supported rust version for the current combine major version. I don't necessarily want to keep this minimum version until the next breaking (major version) but I do try to stick to it unless actually necessary.

For something like documentation though I would rather keep the minimum version as is.

So either skip using matches! or disable doc tests on 1.40 would be acceptable to me.

Reply:
Alright, the next commit attempts to disable doc-tests for 1.40, and should that fail or run into other issues, I will remove `matches!()` all together.